src/components/PdfPageComponent.js

```diff
@@ -60,9 +60,9 @@ class PdfPageComponent {
 
     let page = await this.pdfDoc.getPage(this.pageNum);
 
-    let viewport = this.#setupViewport(page, zoomScale);
+    let viewport = getViewport(page, this.components.canvas, zoomScale);
```

Comment:
What about setupViewport method? It should be deleted.

Reply:
Done